PostBuild.sh

```diff
@@ -51,13 +53,15 @@ function UsageMsg {
       echo "Usage: ${0} [GNU long option] [option] ..."
       echo "  Options:"
       printf '\t%-4s%s\n' '-f' 'Filesystem-type of chroo-devs (e.g., "xfs")'
+      printf '\t%-4s%s\n' '-F' 'Disable FIPS support (NOT IMPLEMENTED)'
       printf '\t%-4s%s\n' '-h' 'Print this message'
       printf '\t%-4s%s\n' '-m' 'Where chroot-dev is mounted (default: "/mnt/ec2-root")'
       printf '\t%-4s%s\n' '-z' 'Initial timezone of build-target (default: "UTC")'
       echo "  GNU long options:"
       printf '\t%-20s%s\n' '--fstype' 'See "-f" short-option'
       printf '\t%-20s%s\n' '--help' 'See "-h" short-option'
       printf '\t%-20s%s\n' '--mountpoint' 'See "-m" short-option'
+      printf '\t%-20s%s\n' '--no-fips' 'See "-F" shortt-option'
```

Comment:
```suggestion
      printf '\t%-20s%s\n' '--no-fips' 'See "-F" short-option'
```

Reply:
Freaking keyboard is dying: either "T" refuses to register or it multiply registers...